README.md

```diff
@@ -86,164 +86,166 @@ Some additionally important notes about how snap works:
+/intel/psutil/cpu/cpu-total/iowait       8
+/intel/psutil/cpu/cpu-total/irq          8
+/intel/psutil/cpu/cpu-total/nice         8
+/intel/psutil/cpu/cpu-total/softirq      8
+/intel/psutil/cpu/cpu-total/steal        8
+/intel/psutil/cpu/cpu-total/stolen       8
+/intel/psutil/cpu/cpu-total/system       8
+/intel/psutil/cpu/cpu-total/user         8
+/intel/psutil/load/load1                 8
+/intel/psutil/load/load15                8
+/intel/psutil/load/load5                 8
+...
+```
+
+NOTE: Plugin bundles are available for convenience in the Snap [GitHub release page](https://github.com/intelsdi-x/snap/releases), for the latest up to date version use the release/download in the [plugin catalog](#plugin-catalog).
 
 ### Running Tasks
-[Tasks](docs/TASKS.md) are most often shared as a Task Manifest and is written in JSON or YAML format.
-
-Create a task manifest file, for example `mock-file.yaml` with following content:
-```yaml
----
-  version: 1
-  schedule:
-    type: "simple"
-    interval: "1s"
-  max-failures: 10
-  workflow:
-    collect:
-      metrics:
-        /intel/mock/foo: {}
-        /intel/mock/bar: {}
-        /intel/mock/*/baz: {}
-      config:
-        /intel/mock:
-          name: "root"
-          password: "secret"
-      process:
-        -
-          plugin_name: "passthru"
-          config:
-            debug: true
-          process: null
-          publish:
-            -
-              plugin_name: "mock-file"
-              config:
-                file: "/tmp/snap_published_mock_file.log"
-                debug: true
-```
 
-and then start the task:
+Tasks can be created directly via `snapctl`, but due to length of task it makes more sense to load from a `Task Manifest` written in JSON or YAML format.
 
+Now, we will download and load the [psutil Task Manifest](examples/psutil-file.yaml):
 ```
-$ cd ~/snap
-$ snapctl task create -t mock-file.yaml
+$ curl https://raw.githubusercontent.com/intelsdi-x/snap/master/examples/tasks/psutil-file.yaml -o /tmp/psutil-file.yaml
+$ snapctl task create -t /tmp/psutil-file.yaml
 Using task manifest to create task
 Task created
 ID: 8b9babad-b3bc-4a16-9e06-1f35664a7679
 Name: Task-8b9babad-b3bc-4a16-9e06-1f35664a7679
 State: Running
 ```
 
-This task generates mock data, "processes" it through a passthrough mechanism and then publishes it to a file. Now with a running task, you should be able to do two things:
+NOTE: in subsequent commands use the task ID from your CLI output, not the example task ID shown below.
 
-See the data that is being published to the file:
+This starts a task collecting metrics via psutil, then publishes the data to a file. To see the data published to the file (CTRL+C to exit):
 ```
 $ tail -f /tmp/snap_published_mock_file.log
-^C
 ```
 
-Or tap into the data that Snap is collecting using the Task ID to watch the task (note: your Task ID will be different):
+Or directly tap into the data stream that Snap is collecting using `snapctl task watch` (
```

Comment:
Should we be explicit about this being the task ID here? i.e. `snapctl task watch <task-id>`